Support max queued_time limit (#25589)
prestodb/presto: presto-docs/src/main/sphinx/admin/properties.rst

This property can be used to configure how long a query runs without contact
from the client application, such as the CLI, before it's abandoned.

The corresponding session property is :ref:`admin/properties-session:\`\`query_client_timeout\`\``.

``query.max-queued-time``

> ^^^^^^^^^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^
```

* **Type:** ``Duration``
* **Default value:** ``100d``

> This property can be used to configure how long a query can be queued before its terminated.

Reviewer: ```suggestion
Use to configure how long a query can be queued before it is terminated.
```
